app/src/main/java/org/piwigo/bg/UploadService.java

```diff
@@ -226,7 +313,7 @@ public void onResponse(@NonNull Call<ImageUploadResponse> call, @NonNull Respons
             }
 
             @Override
-            public void onFailure(@NonNull Call<ImageUploadResponse> call, @NonNull Throwable t) {
+            public void onFailure(Call<ImageUploadResponse> call, Throwable t) {
```

Comment:
why did you remove the @NonNull?

Reply:
The IDE did, I did not really notice.
You want me to add it back ?

Reply:
if NotNull is correct it would be good, but maybe the signature of Callback.onFailure changed?

Reply:
See 8fe2dda